### PR TITLE
SMHE-2159: Object config for Get firmwareversion gas

### PR DESCRIPTION
**Pull request by stefanermens**

### CODE REVIEW

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/Firmware.feature [hunk 1]
```diff
@@ -51,29 +51,62 @@ Feature: SmartMetering Configuration - Firmware
...
+      | Channel                     |             3 |
+      | FirmwareModuleVersionSimple |      19180706 |
     When the get firmware version gas request is received
-      | DeviceIdentification | TEST1027000000002 |
+      | DeviceIdentification | <g-meter> |
     Then the firmware version gas result should be returned
-      | DeviceIdentification | TEST1027000000002 |
-      | SimpleVersionInfo    |          00400011 |
+      | DeviceIdentification | <g-meter> |
+      | SimpleVersionInfo    |  00400011 |
```
Comment (harrymiddelburgetall) [inline integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/Firmware.feature:71]: Make SimpleVersionInfo variable from example or better removed expectedVersion from example does not vary

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/Firmware.feature [hunk 2]
```diff
@@ -51,29 +51,62 @@ Feature: SmartMetering Configuration - Firmware
...
+      | DeviceType                  | SMART_METER_G |
+      | GatewayDeviceIdentification | <e-meter>     |
+      | Channel                     |             3 |
+    When the get firmware version gas request is received
+      | DeviceIdentification | <g-meter> |
+    Then the firmware version gas result should not be returned
+      | DeviceIdentification | <g-meter> |
+
+    Examples:
+      | e-meter           | g-meter           | protocol | version | expectedVersion |
```
Comment (harrymiddelburgetall) [inline integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/Firmware.feature]: expectedVersion is not input for the scenarion

**Review by harrymiddelburgetall — CHANGES_REQUESTED** [review]

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/Firmware.feature (hunk 1 above)
Comment (stefanermens) [reply integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/Firmware.feature:71]: Removed from example

File: integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/Firmware.feature (hunk 2 above)
Comment (stefanermens) [reply integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/configuration/Firmware.feature]: Removed

**Review by harrymiddelburgetall — APPROVED** [review]

**Pull request merged**